### PR TITLE
Link to RestException from RestInterfaceClient

### PR DESCRIPTION
**Pull request by Panke**

### CODE REVIEW

File: web/vibe/web/rest.d [hunk 1]
```diff
@@ -586,6 +586,10 @@ unittest {
 	the matching method style for this. The RestInterfaceClient class will derive from the
 	interface that is passed as a template argument. It can be used as a drop-in replacement
 	of the real implementation of the API this way.
+
+	Non-success:
+		If an request failed or the server returned an non-success status code,
+		an `web.common.RestException` will be thrown.
```
Comment (Geod24) [inline web/vibe/web/rest.d]:
```suggestion
		a `vibe.web.common.RestException` will be thrown.
```

File: web/vibe/web/rest.d [hunk 2]
```diff
@@ -586,6 +586,10 @@ unittest {
 	the matching method style for this. The RestInterfaceClient class will derive from the
 	interface that is passed as a template argument. It can be used as a drop-in replacement
 	of the real implementation of the API this way.
+
+	Non-success:
+		If an request failed or the server returned an non-success status code,
```
Comment (Geod24) [inline web/vibe/web/rest.d]:
```suggestion
		If a request failed or the server returned an non-success status code,
```

You could also mention timeout here.

**Comment by Panke** [comment]
Thanks for the review, will come back to this in the evening.

**Comment by Panke** [comment]
Can we re-run the travis-ci check? Seems to be a fluke, since the error is "could not download diet-ng".

**Comment by s-ludwig** [comment]
Hm, DUB failures have become more frequent in the last months, I think this is related to the HTTP timeout that was added at some point - needs some investigation. I've restarted the check now.

**Comment by Panke** [comment]
Much appreciated

**Pull request merged**